docs/docs/rest-api/public/api/v2/types/pod-instance-status.raml

```diff
@@ -0,0 +1,62 @@
+#%RAML 1.0 DataType
+type: object
+additionalProperties: false
+properties:
+  id:
+    type: !include name.raml
+    description: |
+      Unique ID of this pod instance in the cluster.
+      TODO(jdef) Probably represents the Mesos executor ID.
+  status:
+    type: string
+    enum: [ PENDING, STAGING, STABLE, DEGRADED, TERMINAL ]
```

Comment:
As noted above, the plan for the internal model is to abstract (AppDefinition) task instances and (PodDefinition) instances into one `Instance`. A task can have the following states:
```
Reserved
Created
Error
Failed
Finished
Killed
Killing
Running
Staging
Starting
Unreachable
Gone
Dropped
Unknown
```

For the sake of aligning things in v2, what about simply mapping task status to instance status? @unterstein made plan of how status are aggregated into an aggregate status of a pod instance (e.g., the instance is staging as long as any task is staging, and when all are running, the instance turns running).